Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -61,6 +61,28 @@
   - type: HeldSpeedModifier
   - type: ToggleableClothing
     clothingPrototype: ClothingHeadHelmetHardsuitAtmos
+  - type: GasTank
+    outputPressure: 42.6
+    air:
+      volume: 1.5
+      temperature: 293.15
+      moles:
+        - 0.307706858 # oxygen
```

yamllint

[warning] too few spaces before comment

(comments)